server/flyway/sql/V39__add_client_role.sql

```diff
@@ -0,0 +1,47 @@
+-- Delete any role assignments to the CLIENT initial testing roles
+DELETE FROM app_fam.fam_user_role_xref WHERE role_id IN
+    (SELECT role_id from app_fam.fam_role WHERE application_id IN
+        (SELECT application_id FROM app_fam.fam_application WHERE application_name IN ('CLIENT_DEV', 'CLIENT_TEST', 'CLIENT_PROD'))
+    )
+;
+
+-- Delete previously created CLIENT testing roles for client_dev, client_test and client_prod applications
+DELETE FROM app_fam.fam_role WHERE application_id IN (
+    SELECT application_id FROM app_fam.fam_application WHERE application_name IN ('CLIENT_DEV', 'CLIENT_TEST', 'CLIENT_PROD')
+);
+
+-- Add a role for CLIENT_DEV
+INSERT INTO app_fam.fam_role (
+    role_name,
+    role_purpose,
+    application_id,
+    role_type_code,
+    create_user,
+    create_date
+)
+VALUES ('CLIENT_EDITOR', 'Create and edit client information', (select application_id from app_fam.fam_application where application_name = 'CLIENT_DEV'), 'C', CURRENT_USER, CURRENT_DATE)
+;
+
+-- Add a role for CLIENT_TEST
+INSERT INTO app_fam.fam_role (
```

Comment:
Note that you could have written this as a single insert statement adding the 3 rows, would have been more concise.

Reply:
Even fancier, there's a way to write the insert so that the role name and purpose is only supplied once and a list of application names provided.